src/generators/swift/templates/analytics.swift.hbs

```diff
@@ -0,0 +1,31 @@
+/**
+ * {{autogeneratedFileWarning}}
+ */
+
+#if canImport(Segment)
+import Segment
+#elseif canImport(Analytics)
+import Analytics
+#endif
+
+class TypewriterAnalytics {
+    {{#each tracks}}
+    {{#if description}}
+    /// {{description}}
+    {{/if}}
+    static func {{functionSignature functionName properties false}} {
+        {{functionCall "TypewriterAnalytics" functionName properties "options" "[:]"}}
+    }
+
+    {{#if description}}
```

Comment:
looks like this is called `functionDescription` instead: https://github.com/segmentio/typewriter/search?q=functionDescription

though tbh `description` would be a better name -- feel free to rename, as long you update this in the other templates!